src/fparser/tests/test_parser.py

```diff
@@ -83,115 +92,132 @@ def parse(cls, line, label='', isfree=True, isstrict=False):
     reader.set_mode(isfree, isstrict)
     item = reader.next()
     if not cls.match(item.get_line()):
-        raise ValueError('%r does not match %s pattern'%(line, cls.__name__))
+        raise ValueError('%r does not match %s pattern' % (line, cls.__name__))
     stmt = cls(item, item)
     if stmt.isvalid:
         # Check that we can successfully parse the string representation
         # of the parsed object
-        r = str(stmt)
+        r = str(stmt)  # pylint: disable=invalid-name
```

Comment:
Sorry, I think we should conform to the pep8 variable name requirements. I'm happy for you to not do this in this issue but please remove the pylint directive.